### PR TITLE
[WNMGDS-1722] component content update

### PR DESCRIPTION
**Pull request by scheul93**
## Summary

* Removing 'View Source Link' component from docs site since link to Github will now be in the page header
* Updating links to sketch, github & storybook for all components
* Moving intro sentence into fronmatter for components
* Also found an instance in Mask scss where the value was not being calculated for css vars, so updated that for display on the doc site

## How to test

[Demo site](https://cmsgov.github.io/design-system/branch/WNMGDS-1722/component-content-update)

### CODE REVIEW

**Review by forrestbaer — APPROVED** [review]
Nice! This looks great 👍🏻

File: packages/docs/content/components/accordion.mdx [hunk 1]
```diff
@@ -1,26 +1,31 @@
 ---
 title: Accordion
 status: draft
+intro: An accordion is a list of headers that hide or reveal additional content when selected.
+core:
+  githubLink: design-system/src/components/Accordion/Accordion.tsx
```
Comment (zarahzachz) [inline packages/docs/content/components/accordion.mdx:6]: Curious - should we link to the tsx file or the directory of the component? Or does that even matter?

Comment (scheul93) [reply packages/docs/content/components/accordion.mdx:6]: I could go either way. I think for some, I linked directly to a file and others I linked to the directory. I chose files when the whole directory felt like it had an overwhelming amount of files. But I'm happy to update to whatever folks like

Comment (scheul93) [reply packages/docs/content/components/accordion.mdx:6]: Updating to link to the directory for v1.

**Review by pwolfert — APPROVED** [review]
I ran through a fair number of these pages and checked all the links in the different themes, and I didn't see any mistakes 👍

**Pull request merged**